README.md

```diff
@@ -0,0 +1,54 @@
+# conference
+Project: HTML/CSS & JavaScript capstone project - Conference page.
+ 
+This Project is based on an online website for a conference. 
+A responsive design
+ 
+![screenshot](https://github.com/aliabbani/Conference-page/blob/mobile-first/images/readme.png)
+ 
+ 
+## Built With
+- HTML
+- CSS
+- JavaScript
+- Coffee
+ 
+## Live Demo
+[Live Demo Link](https://aliabbani.github.io/Conference-page/)
```

Comment:
- [ ] OPTIONAL
Your hosted application on the repository does not contain the updated version of you project. Kindly remember to ensure to update the hosted code on Google Pages is updated anytime code change is uploaded to the repository